graphbolt/src/fused_csc_sampling_graph.cc

```diff
@@ -97,12 +97,21 @@ c10::intrusive_ptr<FusedCSCSamplingGraph> FusedCSCSamplingGraph::Create(
   }
   if (node_attributes.has_value()) {
     for (const auto& pair : node_attributes.value()) {
-      TORCH_CHECK(pair.value().size(0) == indptr.size(0) - 1);
+      TORCH_CHECK(
+          pair.value().size(0) == indptr.size(0) - 1,
+          "Expected node_attribute.size(0) and num_nodes to be euqal, "
+          "but node_attribute.size(0) was ",
+          pair.value().size(0), ", and num_nodes was ", indptr.size(0) - 1,
+          ".");
     }
   }
   if (edge_attributes.has_value()) {
     for (const auto& pair : edge_attributes.value()) {
-      TORCH_CHECK(pair.value().size(0) == indices.size(0));
+      TORCH_CHECK(
+          pair.value().size(0) == indices.size(0),
+          "Expected edge_attribute.size(0) and num_edges to be equal, "
+          "but edge_attribute.size(0) was ",
+          pair.value().size(0), ", and num_edges was ", indices.size(0), ".");
```

Comment:
```suggestion
          pair.value().size(0), ", and num_edges was ", NumEdges(), ".");
```

Reply:
It is indeed convenient to make use of existing APIs, but the purpose of this function is to check all arguments necessary to create a FusedCSCSamplingGraph, which means we need to check not only the size of node/edge attributes, but also the size of `indptr` and `indices`. Therefore I insist not to use `NumNodes()` and `NumEdges()` here.

Reply:
https://github.com/dmlc/dgl/blob/af0b63eda6315dcd0db6bdb25f2db84a0db12754/graphbolt/include/graphbolt/fused_csc_sampling_graph.h#L109-L110
https://github.com/dmlc/dgl/blob/af0b63eda6315dcd0db6bdb25f2db84a0db12754/graphbolt/include/graphbolt/fused_csc_sampling_graph.h#L112-L113

Reply:
Either way is fine by me, it is a small nit.